src/scss/admin/constants.scss

```diff
@@ -0,0 +1,2 @@
+$inputSpacingX: 0.75rem;
+$inputSpacingY: 0.5rem;
```

Comment:
[code smell] Might be perfectly OK, might be an indication there is some alternative solution.

Usually .tailwind.scss or .module.scss would compartmentalize these constants if they're specific for a single compoenent.

Alas, if they're shared, (and of course `$kebob-naming--scheme`) they could be put in a specific file as to not bloat `constants.scss`. 

I'm thinking a file like https://github.com/sparkletown/sparkle/blob/rnd/attendee-styles/src/scss/attendee/spacing.scss which is part of a re-organization PR https://github.com/sparkletown/sparkle/pull/2846/files